notebooks/sparse_embedding_retrieval.ipynb

```diff
@@ -0,0 +1,10657 @@
+{
```

Comment:
<p>I didn't understand what you're trying to achieve in this section and how to interpret the outputs. Can you add more explanations here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Improved the explanation.